### ISSUE TITLE
 URL is not a constructor

### ISSUE THREAD

**Issue by arteck** [Apollon77/ioBroker.alexa2#145]
new version 

push is on
and 2-auth off 

```

host.iobroker | 2018-11-17 17:38:21.507 | info | Restart adapter system.adapter.alexa2.0 because enabled
-- | -- | -- | --
host.iobroker | 2018-11-17 17:38:21.507 | error | instance system.adapter.alexa2.0 terminated with code 0 (OK)
host.iobroker | 2018-11-17 17:38:21.507 | error | Caught by controller[0]:     at endReadableNT (_stream_readable.js:974:12)
host.iobroker | 2018-11-17 17:38:21.507 | error | Caught by controller[0]:     at IncomingMessage.emit (events.js:185:7)
host.iobroker | 2018-11-17 17:38:21.507 | error | Caught by controller[0]:     at emitNone (events.js:91:20)
host.iobroker | 2018-11-17 17:38:21.506 | error | Caught by controller[0]:     at IncomingMessage.res.on.e (/opt/iobroker/node_modules/alexa-remote2/alexa-remote.js:653:58)
host.iobroker | 2018-11-17 17:38:21.506 | error | Caught by controller[0]:     at getNotifications (/opt/iobroker/node_modules/alexa-remote2/alexa-remote.js:170:25)
host.iobroker | 2018-11-17 17:38:21.506 | error | Caught by controller[0]:     at prepare (/opt/iobroker/node_modules/alexa-remote2/alexa-remote.js:118:30)
host.iobroker | 2018-11-17 17:38:21.506 | error | Caught  by controller[0]:     at AlexaRemote.initWsMqttConnection  (/opt/iobroker/node_modules/alexa-remote2/alexa-remote.js:543:26)
host.iobroker | 2018-11-17 17:38:21.506 | error | Caught by controller[0]:     at AlexaWsMqtt.connect (/opt/iobroker/node_modules/alexa-remote2/alexa-wsmqtt.js:37:26)
host.iobroker | 2018-11-17 17:38:21.506 | error | Caught by controller[0]:     at new WebSocket (/opt/iobroker/node_modules/alexa-remote2/node_modules/ws/lib/websocket.js:62:20)
host.iobroker | 2018-11-17 17:38:21.505 | error | Caught  by controller[0]:     at WebSocket.initAsClient  (/opt/iobroker/node_modules/alexa-remote2/node_modules/ws/lib/websocket.js:450:17)
host.iobroker | 2018-11-17 17:38:21.504 | error | Caught by controller[0]: TypeError: URL is not a constructor
alexa2.0 | 2018-11-17 17:38:21.471 | error | uncaught exception: URL is not a constructor
alexa2.0 | 2018-11-17 17:38:17.596 | info | starting. Version 1.1.2 in /opt/iobroker/node_modules/iobroker.alexa2, node: v6.12.0
host.iobroker | 2018-11-17 17:38:15.787 | info | instance system.adapter.alexa2.0 started with pid 1530


```

**Comment by arteck** [comment]
with push off works

**Comment by Apollon77** [comment]
yes ... caused by nodejs 6 incompatible change by ws project in a mico release :-(  https://github.com/websockets/ws/issues/1472

**Comment by Apollon77** [comment]
can you please try to downgrade ws lib to 6.1.0 or 6.0.0? Should work then. I wait for a reaction there and maybe release a new version with a pinned ws lib :-(

**Comment by Apollon77** [comment]
@arteck as stated in the WS ticket: can you please update to latest 6.x to verify it happens there too?

**Comment by Apollon77** [comment]
6.14.4 should be current

**Comment by Apollon77** [comment]
PPS ... should be >=6.13.0 ...

**Comment by Apollon77** [comment]
fixed by 1.1.3
